core/commands/follow.go

```diff
@@ -0,0 +1,167 @@
+package commands
+
+import (
+	"errors"
+	"io"
+	"strings"
+	"time"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
```

Comment:
it's a subcommand of name, which is an old style command and needs references to its children.

Reply:
Fair enough. I think once the commands refactor has settled a bit we should update the parent command to use the new cmds lib instead though. For now I think it's okay because that process is still ongoing.